src/backend/tests/test_central_routes.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, 2023 Humanitarian OpenStreetMap Team
+#
+# This file is part of FMTM.
+#
+#     FMTM is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     FMTM is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with FMTM.  If not, see <https:#www.gnu.org/licenses/>.
+#
+"""Tests for central routes."""
+
+import pytest
+
+from app.models.enums import XLSFormType
+
+
+async def test_list_forms(client):
+    """Test get a list of all XLSForms available in FMTM."""
+    response = client.get("/central/list-forms")
+    assert response.status_code == 200
+
+    forms_json = response.json()
+    supported_form_categories = {
+        member.value for member in XLSFormType.__members__.values()
```

Comment:
I'm sure there are simpler ways to get the values here 😄

There is a similar example:
https://github.com/hotosm/fmtm/blob/development/src%2Fbackend%2Fapp%2Fprojects%2Fproject_crud.py#L562

Reply:
Thanks for the example! Applied changes in the latest commit👍